### ISSUE TITLE
henkelBinnacle

### ISSUE THREAD

**Issue by minkull** [researchart/rose6icse#105]
https://github.com/researchart/rose6icse/tree/master/submissions/available/henkelBinnacle
https://github.com/researchart/rose6icse/tree/master/submissions/reusable/henkelBinnacle

## Jordan Henkel

**Corresponding author**

- Email: `jjhenkel@cs.wisc.edu`
- Github Id: `jjhenkel`

## Christian Bird

- Email: `christian.bird@microsoft.com`
- Github Id: `cabird`

## Shuvendu Lahiri

- Email: `shuvendu.lahiri@microsoft.com`
- Github Id: `shuvendu-lahiri`

## Thomas Reps

- Email: `reps@cs.wisc.edu`
- Github Id: `reps-thomas`

## Note to reviewers: these authors want multiple badges
- available
- reusable

**Comment by timm** [comment]
@AnAnonymousReviewer  and @andromeda04 :  I look forward to your reviews

**Comment by JonDoe-ArtifactsTrack** [comment]
Dear Authors,
thank you for your submission. I have followed the steps of your artifact and was able to install it successfully based on the requirements you have listed.
Moreover, once installed, i have managed to execute and reproduce all of the results for each of your RQs.

**Comment by andromeda04** [comment]
The authors apply for the "Reusable" and “Available” badges

- **Reusable**: The artifact is straightforward and work as expected. I could successfully use the artifact to run the experiments and reproduce the experimental results (the three research questions) as shown in the paper.

Some minor comments/suggestions: the authors might consider adding the instructions to install Docker:
$ sudo apt-get update
$ sudo apt-get install docker.io

- **Available**: The authors made their artifact available on GitHub, and also released it on Zenodo to obtain a DOI. 

**Recommendation**: I recommend both badges for this paper.

**Comment by jjhenkel** [comment]
Thanks for the reviews!

@andromeda04, I agree that instructions to install Docker would be helpful. I've added some links for installing Docker on Windows/Mac/Ubuntu to the README in this commit: https://github.com/jjhenkel/binnacle-icse2020/commit/392f4f8ac3481e802a2d8b275b43df622cd1785e

I've just linked Docker's own installation instructions as the commands you listed may not work for other Linux distros.
